src/settings/CustomFieldsSettings.js

```diff
@@ -18,7 +18,7 @@ const CustomFieldsSettings = () => {
 
   const permissions = {
     canView: stripes.hasPerm('ui-orders.settings.custom-fields.view'),
-    canEdit: stripes.hasPerm('ui-orders.settings.custom-fields.edit'),
+    editable: stripes.hasPerm('ui-orders.settings.custom-fields.edit'),
```

Comment:
@alisher-epam This change seems to cause [UITEN-293](https://folio-org.atlassian.net/browse/UITEN-293)

Reply:
Will fix it on the next PR https://github.com/folio-org/ui-orders/pull/1610